docs/tutorials/podman_pasta_ipv6_with_snid_ipv4.md

```diff
@@ -0,0 +1,761 @@
+    restart: "unless-stopped"
+    security_opt:
+      - no-new-privileges:true
+      - label=type:container_runtime_t
+    ports:
+      - target: 80
+        host_ip: "[${APPLICATION_IPV6_ADDRESS}]"
+        published: 80
+        protocol: tcp
+      - target: 443
+        host_ip: "[${APPLICATION_IPV6_ADDRESS}]"
+        published: 443
+        protocol: tcp
+      - target: 443
+        host_ip: "[${APPLICATION_IPV6_ADDRESS}]"
+        published: 443
+        protocol: udp
+    network_mode: "pasta:--ipv6-only"
+    volumes:
+ #     - /run/user/1001/podman/podman.sock:/var/run/docker.sock:rw,z
+      - ./Caddyfile:/etc/caddy/Caddyfile:ro,z
+      - ~/containers/data/whoami-caddy:/data:rw,z
+      - ~/containers/log/whoami-caddy:/var/log:rw,z
+      - ~/containers/config/whoami-caddy:/config:rw,z
+      - ~/containers/certificates/letsencrypt:/certificates:ro,z
+    environment:
+      - CADDY_DOCKER_CADDYFILE_PATH=/etc/caddy/Caddyfile
+
+  # Proxy to container
+  whoami-application:
+    image: traefik/whoami
+    pull_policy: "missing"
+    container_name: whoami-application
+    restart: "unless-stopped"
+    network_mode: "service:whoami-caddy"
+    environment:
+      - WHOAMI_PORT_NUMBER=8080
+```
+
+## Compose with only Pasta Line
+With this Method, `podman ps` will NOT show the Open Ports in its Output.
+
+In fact, `podman inspect <container>` will NOT list the IP Addresses nor where the Ports are bound to.
+
+This works correctly, but the only way to examine if the Port is used is to run `ss -nlt6`.
+
+```
+services:
+  whoami-caddy:
+    image: caddy:latest
+    #image: lucaslorentz/caddy-docker-proxy:2.9-alpine
+    pull_policy: "missing"
+    container_name: whoami-caddy
+    restart: "unless-stopped"
+    security_opt:
+      - no-new-privileges:true
+      - label=type:container_runtime_t
+    network_mode: "pasta:--ipv6-only,-t,${APPLICATION_IPV6_ADDRESS}/80,-t,${APPLICATION_IPV6_ADDRESS}/443,-u,${APPLICATION_IPV6_ADDRESS}/443"
+    volumes:
+ #     - /run/user/1001/podman/podman.sock:/var/run/docker.sock:rw,z
+      - ./Caddyfile:/etc/caddy/Caddyfile:ro,z
+      - ~/containers/data/whoami-caddy:/data:rw,z
+      - ~/containers/log/whoami-caddy:/var/log:rw,z
+      - ~/containers/config/whoami-caddy:/config:rw,z
+      - ~/containers/certificates/letsencrypt:/certificates:ro,z
+    environment:
+      - CADDY_DOCKER_CADDYFILE_PATH=/etc/caddy/Caddyfile
+
+  # Proxy to container
+  whoami-application:
+    image: traefik/whoami
+    pull_policy: "missing"
+    container_name: whoami-application
+    restart: "unless-stopped"
+    network_mode: "service:whoami-caddy"
+    environment:
+      - WHOAMI_PORT_NUMBER=8080
+```
+
+> **Warning**  
+> 
+> Netstat is deprecated. `netstat -an | grep -i listen` will NOT return the correct IPv6 Addresses in most cases, because it truncates the Output. `netstat -Wan | grep -i listen` will return the full IPv6 Address wthout truncating it, but you should really be using `ss -nlt6` instead.
```

Comment:
1. `netstat ... | grep listen` is unnecessary. `netstat -l` does the same
2. just explain to use `ss -tulpn6` and possible an example output.

Reply:
I didn't know that Option. However, it is NOT exactly the same thing. `netstat -l` also lists the UNIX Sockets, which is not what I (and I guess most people) are interested in.

`netstat -an | grep -E "^tcp|tcp6|udp|udp6"` or `netstat -an | grep -Ev "^unix"` might limit the Results to what we look for. Anyways, as discussed with sbrivio yesterday, adding `-W` is mandatory (otherwise incorrect IP Addresses are listed).

But I see that `netstat` has an Option to limit the Type of Socket, so maybe `netstat -Wan -t -u` is what we want :smiley: (or just `netstat -Wan -t` for TCP).

Reply:
I changed the Warning into a small Debugging Section in the new Version (I'll commit in a few Minutes).

## Debugging Port Binding
Note that `netstat` is deprecated. You **really** should be using `ss -nlt` instead.

That being said, `netstat -an -t` will NOT return the correct IPv6 Addresses in most cases, because it truncates the Output. `netstat -Wan -t` (Note the `-W` additional Argument) will return the full IPv6 Address wthout truncating it.

To check that The Services bind to the Correct IP Address and Port:
- IPv4 + IPv6: `ss -nlt` (or `netstat -Wan -t`)
- IPv4 Only: `ss -nlt4` (or `netstat -Wan -t4`)
- IPv6 Only: `ss -nlt6` (or `netstat -Wan -t6`)

Reply:
Commit pushed. I also added an example output :+1:. Please open a new Conversation if still not OK. I'll close this one for now.